roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -79,6 +79,7 @@ def _format_failure(failure):
         (u'Host', host),
         (u'Play', play),
         (u'Task', task),
+        (u'Changed', stringc(result._result.get('changed', u'???'), C.COLOR_CHANGED)),
```

Comment:
This output was focused on listing a summary of failures. Not sure about changed True/False being valuable there.